dima/react-movies/src/App.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { Component } from "react";
+import MainPage from "./pages/mainpage";
+import Movies from "./api/movies";
+import "./App.css";
+import MoviePage from "./pages/moviepage";
+import MovieInterface from "./interfaces/movieInterface";
+
+interface AppState {
+  movie: MovieInterface;
+  movies: MovieInterface[];
+  searchedMovies: null | MovieInterface[];
+  currentSortType: string;
+}
+class App extends Component<{}, AppState> {
+  state: AppState = {
+    movie: Movies[0],
+    movies: Movies,
+    searchedMovies: null,
+    currentSortType: "release date",
```

Comment:
strings such "release date" better to move to enum.

Reply:
fixed